### PR TITLE
FIXES #5783 Multiple meta mofs

### LINKED ISSUE #5783: Two Localhost mof files in config folder  localhost.mof & mycomputername.mof
From the documentation I can't see what would happen if I ran:
Set-DscLocalConfigurationManager -Path "C:\DSC\Configurations\"  where I have two mof files, one with the actual hostname and the other with 'localhost'....Do they both get applied?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 1c233a73-f2d2-a27e-4c0f-baf5e4076b5f
* Version Independent ID: 2b81adca-6b3c-ef25-5c25-c543c5ee04bd
* Content: [Set-DscLocalConfigurationManager (PSDesiredStateConfiguration)](https://docs.microsoft.com/en-us/powershell/module/psdesiredstateconfiguration/set-dsclocalconfigurationmanager?view=powershell-5.1)
* Content Source: [reference/5.1/PSDesiredStateConfiguration/Set-DscLocalConfigurationManager.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/5.1/PSDesiredStateConfiguration/Set-DscLocalConfigurationManager.md)
* Product: **powershell**
* Technology: **powershell-cmdlets**
* GitHub Login: @nitinbps
* Microsoft Alias: **nitingup**

### PR DESCRIPTION
**Pull request by chasewilson**
# PR Summary
Customer had some confusion about what would happen if multiple meta mofs were available to apply to a machine. Documentation was updated for MD formatting and to add a note to address issue

## PR Context

Fixes #5783 
Fixes [AB#1713210](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1713210)

Select the type(s) of documents being changed.

**Cmdlet reference & about_ topics**
- [ ] Version 7.x preview content
- [ ] Version 7.0 content
- [ ] Version 6 content
- [ ] Version 5.1 content

**Conceptual articles**
- [ ] Fundamental conceptual articles
- [ ] Script sample articles
- [x] DSC articles
- [ ] Gallery articles
- [ ] JEA articles
- [ ] WMF articles
- [ ] SDK articles

## PR Checklist

- [x] I have read the [contributors guide][contrib] and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords][key].
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[WIP]` to the beginning of the
    title and remove the prefix when the PR is ready.

[contrib]: https://docs.microsoft.com/powershell/scripting/community/contributing/overview
[key]: https://help.github.com/en/articles/closing-issues-using-keywords

### CODE REVIEW

File: reference/5.1/PSDesiredStateConfiguration/Set-DscLocalConfigurationManager.md [hunk 1]
```diff
@@ -8,65 +8,77 @@ online version: https://docs.microsoft.com/powershell/module/psdesiredstateconfi
...
 ### Example 1: Apply LCM settings
+
 ```
-PS C:\> Set-DscLocalConfigurationManager -Path "C:\DSC\Configurations\"
+Set-DscLocalConfigurationManager -Path "C:\DSC\Configurations\"
 ```
 
-This command applies the LCM settings from C:\DSC\Configurations\ to the targeted nodes.
-After receiving the settings, LCM processes them.
+This command applies the LCM settings from C:\DSC\Configurations\ to the targeted nodes. After
```
Comment (sdwheeler) [inline reference/5.1/PSDesiredStateConfiguration/Set-DscLocalConfigurationManager.md:48]: Put the file path in backticks.

File: reference/5.1/PSDesiredStateConfiguration/Set-DscLocalConfigurationManager.md [hunk 2]
```diff
@@ -8,65 +8,77 @@ online version: https://docs.microsoft.com/powershell/module/psdesiredstateconfi
...
-Alternatively, create an array of CIM sessions to apply the cmdlet to multiple specified computers.
+This example applies LCM settings to a computer and applies the settings. The example creates a CIM
+session for a computer named Server01 for use with the cmdlet. Alternatively, create an array of CIM
+sessions to apply the cmdlet to multiple specified computers.
 
-The first command creates a CIM session by using the **New-CimSession** cmdlet, and then stores the **CimSession** object in the $Session variable.
-The command prompts you for a password.
-For more information, type `Get-Help New-CimSession`.
+The first command creates a CIM session by using the `New-CimSession` cmdlet, and then stores the
+**CimSession** object in the $Session variable. The command prompts you for a password. For more
```
Comment (sdwheeler) [inline reference/5.1/PSDesiredStateConfiguration/Set-DscLocalConfigurationManager.md:67]: Put the variable in backticks

File: reference/5.1/PSDesiredStateConfiguration/Set-DscLocalConfigurationManager.md [hunk 3]
```diff
@@ -8,65 +8,77 @@ online version: https://docs.microsoft.com/powershell/module/psdesiredstateconfi
...
+information, type `Get-Help New-CimSession`.
 
-The second command applies LCM settings for the targeted node from C:\DSC\Configurations\ to the computer identified by the **CimSession** objects stored in the $Session variable.
-In this example, the $Session variable contains a CIM session only for the computer named Server01.
-The command applies the settings.
-After receiving the settings, LCM processes them.
+The second command applies LCM settings for the targeted node from C:\DSC\Configurations\ to the
```
Comment (sdwheeler) [inline reference/5.1/PSDesiredStateConfiguration/Set-DscLocalConfigurationManager.md:70]: Put the file path in backticks.

File: reference/5.1/PSDesiredStateConfiguration/Set-DscLocalConfigurationManager.md [hunk 4]
```diff
@@ -8,65 +8,77 @@ online version: https://docs.microsoft.com/powershell/module/psdesiredstateconfi
...
+computer identified by the **CimSession** objects stored in the $Session variable. In this example,
+the $Session variable contains a CIM session only for the computer named Server01. The command
```
Comment (sdwheeler) [inline reference/5.1/PSDesiredStateConfiguration/Set-DscLocalConfigurationManager.md:72]: Put the variable in backticks.

File: reference/5.1/PSDesiredStateConfiguration/Set-DscLocalConfigurationManager.md [hunk 5]
```diff
@@ -8,65 +8,77 @@ online version: https://docs.microsoft.com/powershell/module/psdesiredstateconfi
...
 
 ## PARAMETERS
 
 ### -CimSession
-Runs the cmdlet in a remote session or on a remote computer.
-Enter a computer name or a session object, such as the output of a [New-CimSession](https://docs.microsoft.com/powershell/module/cimcmdlets/new-cimsession) or [Get-CimSession](https://docs.microsoft.com/powershell/module/cimcmdlets/get-cimsession) cmdlet.
+
+Runs the cmdlet in a remote session or on a remote computer. Enter a computer name or a session
+object, such as the output of a [New-CimSession](https://docs.microsoft.com/powershell/module/cimcmdlets/new-cimsession)
+or [Get-CimSession](https://docs.microsoft.com/powershell/module/cimcmdlets/get-cimsession) cmdlet.
```
Comment (sdwheeler) [inline reference/5.1/PSDesiredStateConfiguration/Set-DscLocalConfigurationManager.md:81]: Shorten the URLs to be site-relative

File: reference/5.1/PSDesiredStateConfiguration/Set-DscLocalConfigurationManager.md [hunk 6]
```diff
@@ -130,9 +145,10 @@ Accept wildcard characters: False
 ```
 
 ### -Path
-Specifies a file path of a folder that contains configuration settings files.
-The cmdlet publishes and applies these LCM settings to computers that have settings files in the specified path.
-Each target node must have a settings file of the following format: NetBIOS Name.meta.mof.
+
+Specifies a file path of a folder that contains configuration settings files. The cmdlet publishes
+and applies these LCM settings to computers that have settings files in the specified path. Each
+target node must have a settings file of the following format: NetBIOS Name.meta.mof.
```
Comment (sdwheeler) [inline reference/5.1/PSDesiredStateConfiguration/Set-DscLocalConfigurationManager.md:151]: Put the file path in backticks.

File: reference/5.1/PSDesiredStateConfiguration/Set-DscLocalConfigurationManager.md [hunk 7]
```diff
@@ -205,9 +229,6 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 ## RELATED LINKS
 
-[Windows PowerShell Desired State Configuration Overview](/powershell/scripting/dsc/overview/dscforengineers)
+[Windows PowerShell Desired State Configuration Overview](/reference/docs-conceptual/dsc/overview/DscForEngineers.md)
```
Comment (sdwheeler) [inline reference/5.1/PSDesiredStateConfiguration/Set-DscLocalConfigurationManager.md:232]: Remove the file extension from the URL

File: reference/5.1/PSDesiredStateConfiguration/Set-DscLocalConfigurationManager.md (hunk 2 above)
Comment (chasewilson) [reply reference/5.1/PSDesiredStateConfiguration/Set-DscLocalConfigurationManager.md:67]: Fixed

File: reference/5.1/PSDesiredStateConfiguration/Set-DscLocalConfigurationManager.md (hunk 3 above)
Comment (chasewilson) [reply reference/5.1/PSDesiredStateConfiguration/Set-DscLocalConfigurationManager.md:70]: Fixed

File: reference/5.1/PSDesiredStateConfiguration/Set-DscLocalConfigurationManager.md (hunk 4 above)
Comment (chasewilson) [reply reference/5.1/PSDesiredStateConfiguration/Set-DscLocalConfigurationManager.md:72]: Fixed

File: reference/5.1/PSDesiredStateConfiguration/Set-DscLocalConfigurationManager.md (hunk 7 above)
Comment (chasewilson) [reply reference/5.1/PSDesiredStateConfiguration/Set-DscLocalConfigurationManager.md:232]: Fixed

File: reference/5.1/PSDesiredStateConfiguration/Set-DscLocalConfigurationManager.md (hunk 1 above)
Comment (chasewilson) [reply reference/5.1/PSDesiredStateConfiguration/Set-DscLocalConfigurationManager.md:48]: Fixed

File: reference/5.1/PSDesiredStateConfiguration/Set-DscLocalConfigurationManager.md (hunk 5 above)
Comment (chasewilson) [reply reference/5.1/PSDesiredStateConfiguration/Set-DscLocalConfigurationManager.md:81]: Fixed

File: reference/5.1/PSDesiredStateConfiguration/Set-DscLocalConfigurationManager.md (hunk 6 above)
Comment (chasewilson) [reply reference/5.1/PSDesiredStateConfiguration/Set-DscLocalConfigurationManager.md:151]: Fixed

**Pull request merged**